tests/transforms/test_broadcast_expand.py

```diff
@@ -107,33 +94,23 @@ def test_expansion(self, params, size, obs, exp_fn):
         result = fn(qml.execute(tapes, dev, None))
         expected = exp_fn(*params)
 
-        if len(tape.measurements) > 1 and size == 1:
-            expected = expected.T
```

Comment:
The extraction of the first entry of `out` in the `exp_fn`s takes care of this already. There is nothing to transpose because in this case there only is one axis in `expected`.